Hippo.Core/Services/NotificationService.cs

```diff
@@ -62,6 +62,10 @@ public async Task<bool> AccountDecision(Request request, bool isApproved, string
                 var group = await _dbContext.Groups.Where(g => g.ClusterId == request.ClusterId && g.Name == request.Group).SingleAsync();
                 var requestData = request.GetAccountRequestData();
 
+                var message = string.IsNullOrWhiteSpace(details) && !isApproved
+                    ? "Your account request has been rejected. If you believe this was done in error, please contact your sponsor directly. You will need to submit a new request, but contact your sponsor first."
+                    : details;
```

Comment:
this is pretty confusing to read.  Why not just set to `details` and then overwrite if !Approved.  Or some function to get message from details